App/Targets/three-dollar-in-my-pocket/Sources/domain/shared-view/SocialSigninButton.swift

```diff
@@ -70,9 +70,9 @@ final class SocialSigninButton: BaseView {
             
         case .apple:
             self.containerButton.accessibilityLabel = "sign_in_with_apple".localized
-            self.containerButton.backgroundColor = .white
+            self.containerButton.backgroundColor = .black
```

Comment:
컬러도 v4부터는 디자인 시스템에서 관리하려해용!
```suggestion
            self.containerButton.backgroundColor = DesignSystemAsset.Colors.systemBlack.color
```

Reply:
헉 black도 있는지 몰랐어요 
디자인 시스템 한번 더 확인해보겠습니다 🙌🏻🥹

Reply:
네넹!!! 제가 정의했다고 말씀 안드려서 그럴겁니다 ㅠㅠ 아마 피그마 v4에 네이밍된 컬러는 전부 디자인시스템에 정의해두었어요!  
정의가 안되어있는 컬러가 있다면 디자인 시스템에 새로 추가해야합니다!